Pkgfile

```diff
@@ -3,7 +3,7 @@
 format: v1alpha2
 
 vars:
-  TOOLS_IMAGE: ghcr.io/siderolabs/tools:v1.0.0-1-g4c77d96
+  TOOLS_IMAGE: ghcr.io/siderolabs/tools:v1.0.0-2-g5a97af0
```

Comment:
Hmm - is this correct version? https://github.com/siderolabs/tools/pkgs/container/tools/versions has no such versions :\

Reply:
just not published yet :wink: